mep/people/forms.py

```diff
@@ -110,22 +114,39 @@ class MemberSearchForm(FacetForm):
     }))
     membership_dates = RangeField(label='Membership Dates', required=False,
         widget=RangeWidget(attrs={'size': 4}))
+    birth_year = RangeField(required=False,
+        widget=RangeWidget(attrs={'size': 4}))
 
-    def set_membership_dates_placeholder(self, min_year, max_year):
-        '''Set the min, max, and placeholder values for
-        :class:`mep.common.forms.RangeWidget` associated with membership_dates.'''
-
-        start_widget, end_widget = \
-            self.fields['membership_dates'].widget.widgets
-
-        # set placeholders for widgets individually
-        start_widget.attrs['placeholder'] = min_year
-        end_widget.attrs['placeholder'] = max_year
-        # valid min and max for both via multiwidget
-        self.fields['membership_dates'].widget.attrs.update({
-            'min': min_year,
-            'max': max_year
-        })
+    def set_daterange_placeholders(self, min_max_conf):
+        '''Set the min, max, and placeholder values for all fields associated
+        with :class:`mep.common.forms.RangeWidget`.
```

Comment:
maybe clearer to say something like "all `RangeField` instances"

Reply:
Done.

Reply:
I don't think I commented on this on the first round and probably not worth changing at this point, but calling it something like `range_maxmin` instead of `range_placeholder` would be more semantic/informative.